x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/workspace_panel.test.tsx

```diff
@@ -452,32 +455,47 @@ describe('workspace_panel', () => {
 
   describe('suggestions from dropping in workspace panel', () => {
     let mockDispatch: jest.Mock;
+    let frame: jest.Mocked<FramePublicAPI>;
+
+    const draggedField: IndexPatternField = {
+      name: '@timestamp',
+      type: 'date',
+      searchable: false,
+      aggregatable: false,
+    };
```

Comment:
I thought you wrapped this in an object that also contained index pattern?

Reply:
Right, this shouldn't be here. The object passed in doesn't matter because everything is mocked.